ops/review-deploy.tmpl.yaml

```diff
@@ -0,0 +1,220 @@
+replicaCount: 2
+
+livenessProbe:
+  enabled: false
+readinessProbe:
+  enabled: false
+
+brandingVolume:
+  storageClass: nfs
+derivativesVolume:
+  storageClass: nfs
+uploadsVolume:
+  storageClass: nfs
+
+extraVolumeMounts:
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/imports
+    subPath: imports
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/exports
+    subPath: exports
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/public/system
+    subPath: public-system
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/public/uploads
+    subPath: public-uploads
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/network_files
+    subPath: network-files
+
+ingress:
+  enabled: true
+  hosts:
+    - host: hyku-$CI_MERGE_REQUEST_ID.example.com
+      paths:
+        - path: /
+    - host: "*.hyku-$CI_MERGE_REQUEST_ID.example.com"
+      paths:
+        - path: /
+
+extraEnvVars:
```

Comment:
as follow up to this, let's look into getting some of this into the Hyrax chart as a courtesy to Hyku users. especially the `SETTINGS__` stuff is harmless to non hyku users but in some cases duplicates other variables used by the engine.

Reply:
I think we are going to go the opposite way - one of the very next steps is to 100% redo the way settings work in Hyku and I think syncing them to use the Hyrax ones where we can and simplifying to parts where we can not can happen at that time.

Reply:
oh, that's great.